.github/workflows/pr-go-lint.yml

```diff
@@ -8,12 +8,6 @@ env:
   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
 on:
-  # push:
-  #   tags:
-  #     - v*
-  #   branches:
-  #     - master
-  #     - main
   pull_request:
```

Comment:
We can add file-based filters on this (and everywhere else) now.

Reply:
maybe future PR